mitodl/ocw-hugo-themes
# fix: video tab section toggle

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/537

#### What's this PR do?
- Creates a partial for video tab section which can expand/collapse
- Uses this partial to render:
    - Transcript section (if any)
    - Related resources section (if any)
    - Optional tab section (if any)
- Fixes the issue mentioned in the ticket
- Makes the expand/collapse smooth
- Some other CSS tweaks

#### How should this be manually tested?
- Checkout this branch
- Build courses(s) locally which has lecture videos along with transcript, Related resources, and Optional tab sections
- Test the expand/collapse widget/functionality for:
    - Transcript section
    - Related resources section
    - Optional tab section
- Verify that everything works fine.

#### Screenshots (if appropriate)
<img width="674" alt="image" src="https://user-images.githubusercontent.com/93309234/160366336-9930d7a6-9ef6-443c-b8c8-a44164702349.png">

<img width="635" alt="image" src="https://user-images.githubusercontent.com/93309234/160366366-c247b1be-99b2-4431-a11b-9df36a2c01fd.png">

<img width="613" alt="image" src="https://user-images.githubusercontent.com/93309234/160366427-a54bf927-7880-4dc0-9486-8226decabce0.png">
